### PR TITLE
chore: remove the concept of target and host platforms

### PR DESCRIPTION
**Pull request by jviotti**
Its very unlikely that we will ever support cross platform builds. For
that reason, let's simplify the whole Makefile by removing the concept
of target and host platforms.

Signed-off-by: Juan Cruz Viotti <jv@jviotti.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -178,10 +172,10 @@ TARGET_ARCH_DEBIAN = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET
 TARGET_ARCH_REDHAT = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t redhat)
 TARGET_ARCH_APPIMAGE = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t appimage)
 TARGET_ARCH_ELECTRON_BUILDER = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t electron-builder)
-TARGET_PLATFORM_PKG = $(shell ./scripts/build/target-convert.sh -r $(TARGET_PLATFORM) -t pkg)
+PLATFORM_PKG = $(shell ./scripts/build/target-convert.sh -r $(PLATFORM) -t pkg)
```
Comment (lurch) [inline Makefile]: Probably makes sense to rename `target-convert.sh` to `platform-convert.sh` ?

**Comment by lurch** [comment]
Oh, is cross-platform builds not a feature that `electron-builder` supports?

**Comment by jviotti** [comment]
@lurch Up to a certain point. See https://github.com/electron-userland/electron-builder/wiki/Multi-Platform-Build

> Don't expect that you can build app for all platforms on one platform
> - If your app has native dependencies, it can be compiled only on the target platform. prebuild is a solution, but most node modules don't provide prebuilt binaries.
> - macOS Code Signing works only on macOS. Cannot be fixed.

Both points affect us, plus I don't think its a battle worth fighting.

**Comment by lurch** [comment]
Fair enough :+1: 
Rename `target-convert.sh` and I'll approve ;)

File: Makefile (hunk 1 above)
Comment (jviotti) [inline Makefile]: Done!

**Comment by jviotti** [comment]
Cool, I just pushed that change :)

**Comment by jhermsmeier** [comment]
> macOS Code Signing works only on macOS. Cannot be fixed.

This sounds like something that can be fixed by some mild reverse engineering :)

**Comment by lurch** [comment]
I think getting the MacOS CLI code-signed via `pkg` would be more important than fixing `electron-builder` to be able to code-sign MacOS binaries when running on Linux ;)

**Comment by jhermsmeier** [comment]
Yeah, but this sure sounds like a fun spare-time project to me :)

**Comment by lurch** [comment]
As long as Apple doesn't come after you under the DMCA ;-)

**Comment by jhermsmeier** [comment]
Reverse engineering is legal, unless stuff has been copied from whatever it is your tearing apart, afaik. Also... I'm in Germany, not the US ;P

**Comment by lurch** [comment]
:de: :tada:

**Pull request merged**